features/fixtures/mazerunner/jvm-scenarios/src/main/java/com/bugsnag/android/mazerunner/scenarios/InternalReportScenario.kt

```diff
@@ -27,20 +27,12 @@ internal class InternalReportScenario(
             storageManager.setCacheBehaviorGroup(errDir, true)
             storageManager.setCacheBehaviorTombstone(errDir, true)
         }
-
-        if (eventMetadata != "non-crashy") {
-            disableAllDelivery(config)
-        } else {
-            val files = errDir.listFiles()
-            files.forEach { it.writeText("{[]}") }
```

Comment:
This block needs to be reinstated for the scenario to work

Reply:
Added back in, in a new `startBugsnag` method.